tests/core/service/presence/presence_service_test.py

```diff
@@ -36,19 +41,19 @@ def fixture_presence_service(mocked_presence_api_client, auth_session):
     return PresenceService(mocked_presence_api_client, auth_session)
 
 
-@pytest.fixture(name="presence_payload")
-def fixture_presence_payload():
-    return object_from_json_relative_path("presence_response/presence_payload.json")
 
 
 @pytest.mark.asyncio
-async def test_get_presence(presence_service, mocked_presence_api_client, presence_payload):
-    mocked_presence_api_client.v2_user_presence_get.return_value = presence_payload.user_presence
+async def test_get_presence(presence_service, mocked_presence_api_client):
+    mocked_presence_api_client.v2_user_presence_get.return_value = deserialize_object(V2Presence,
+                                                                                      "{\"category\":\"AVAILABLE\","
+                                                                                      "\"userId\":14568529068038,"
+                                                                                      "\"timestamp\":1533928483800}")
 
     presence = await presence_service.get_presence()
 
     assert presence.category == "AVAILABLE"
-    assert presence.userId == 14568529068038
+    assert presence.user_id == 14568529068038
```

Comment:
is it related to the change here?

Reply:
yes

Reply:
I didn't see catch the usage of this in the previous ticket.
The last commit removes all remaining usage of objects directly created from Json.